src/xdebugConnection.ts

```diff
@@ -823,3 +827,33 @@ export class Connection extends DbgpConnection {
         return new EvalResponse(await this._enqueueCommand('eval', undefined, expression), this)
     }
 }
+/**
+ * This class represents a connection to Dbgp proxy server for multi user debugging
+ */
+export class DbgpProxyConnection extends Connection {
```

Comment:
So a proxy connection basically supports all commands that a normal connection supports, plus `proxyinit` and `proxystop`. Is that correct?

Reply:
correct